include/functions.h

```diff
@@ -3932,8 +3932,8 @@ void func_801A4D00(void);
 // void func_801A4EB0(void);
 // void func_801A4EB8(void);
 // void func_801A4FD8(void);
-// void func_801A5080(void);
-// void func_801A5100(void);
+UNK_RET func_801A5080(UNK_TYPE arg0);
+UNK_TYPE func_801A5100(void);
```

Comment:
```suggestion
void func_801A5080(u16 arg0);
u16 func_801A5100(void);
```
We know the prototype for these function from our voice recognition unit WIP